### DISCUSSION TITLE
Multiple images for recipe or user images

### DISCUSSION THREAD

**Discussion by wtal710174** [TandoorRecipes/recipes#1762] [Ideas]
First of all I would like to express how exited I am about this project. I was thinking about creating something like this on my own and eventually, before being able to collect enough energy to start, I find this awesome recipe database. 

One idea:
What about multiple images for recipes? The images could be created, while adding a recipe using the picture drag and drop area in order to show multiple steps easily (After adding small icons or a list of the pictures could be shown to allow removal)  or later by users that would like to show their result in the comments. As of now the comment does accept text only.

**Comment by vabene1111** [comment]
thanks for your feedback :) it is planned to improve the ability to use images in recipes, it has just not yet been implemented, there is an issue somewhere i cannot find right now

**Comment by eydrimm** [comment]
I would also like to say a big thank you for this great app! 

I wanted to ask if there has been any progress in finding a solution to the problem that is preventing more than one image from being uploaded in a recipe. 
Alternatively, could it be an option/workaround to enable image-upload in the comments additionally to text? This way it would also be possible to have more images (at least in the comments).

**Reply by smilerz** [reply]
You can add images to the steps.

**Reply by ISeeTWizard** [reply]
> You can add images to the steps.

You write that but I don't have that option - how do you did that?
I use a simple docker configuration (just started yesterday with it)
Or ist the option with the URL where I need to first upload the images?

**Comment by eydrimm** [comment]
Thank you @smilerz for your quick reply. I will try this workaround

**Comment by ISeeTWizard** [comment]
I was looking for exactly the same feature and already thought I missed something...
So it's simply not yet available as I see

**Comment by whalebeach** [comment]
I am confused, can you add images to the steps in the latest v2.2.7? I couldn't find an option.

If you indeed cannot, it would be an awesome feature since photos help a lot with those little tricks you learn on the way (think "how should the dough look" or "how do I stack the ingredients in this particular casserole dish")

**Reply by vabene1111** [reply]
just add a file to the step
